src/StackExchange.Redis/Profiling/IProfiledCommand.cs

```diff
@@ -27,11 +27,26 @@ public interface IProfiledCommand
         /// </summary>
         string Command { get; }
 
+        /// <summary>
+        /// The name of the script when EVAL command is issued.
+        /// </summary>
+        string Script { get; }
```

Comment:
it this is the name of a script, perhaps it might be called `ScriptName`?

Reply:
It's the actual script, I fixed up the comment.